### PR TITLE
Corrected typos

### PR DESCRIPTION
**Pull request by DataStrategist**
I have just gone through this guide to ensure functionality (for the first part), so am reasonably confident in mods, but please review carefully.

### CODE REVIEW

File: source/documentation/other-topics/route53-zone.html.md.erb [hunk 1]
```diff
@@ -110,7 +116,7 @@ resource "kubernetes_secret" "example_route53_zone_sec" {
 
 ## Creating the resource
 
-Make sure to replace the placeholders and example values above with relevant ones. If you are referring from variables in variables.tf, replace with `var.VARIABLE NAME`. Commit your changes to a branch and raise a pull request.
+Commit your changes to a new branch and raise a pull request.
```
[the remark this answers is not in the document]
Comment (DataStrategist) [reply source/documentation/other-topics/route53-zone.html.md.erb:119]: I had moved this line up, so you have the info before the code... if you want to keep it down here, you can delete it, but I'm happy either way.

File: source/documentation/other-topics/route53-zone.html.md.erb [hunk 2]
```diff
@@ -13,7 +13,7 @@ This short guide will run through the process of creating a [Route 53 Hosted Zon
 
 ## Pre-Requisites
 
-This guide assumes you have an environment already created in the `Live` cluster and defined in the [cloud-platform-environments repository][env-repo].
+This guide assumes you have an environment already created in the `Live` cluster and defined in the [cloud-platform-environments repository][env-repo]. If you have a service runnig on a `*.apps.live.cloud-platform.service.justice.gov.uk/` URL, then you are on the Live cluster already.
```
[the remark this answers is not in the document]
Comment (DataStrategist) [reply source/documentation/other-topics/route53-zone.html.md.erb:16]: of course :)

File: source/documentation/other-topics/route53-zone.html.md.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (poornima-krishnasamy) [reply source/documentation/other-topics/route53-zone.html.md.erb:119]: oh yes I didnot see that.

**Review by jasonBirchall — APPROVED** [review]

**Comment by jasonBirchall** [comment]
The PR is appreciated. Thanks, @DataStrategist

**Pull request merged**